sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FieldValue.java

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The FieldValue model.
+ */
+@Fluent
+public final class FieldValue {
+    private final FieldValueType type;
+    private Map<String, FormField> formFieldMap;
+    private List<FormField> formFieldList;
+    private Float formFieldFloat;
+    private Integer formFieldInteger;
+    private LocalDate formFieldDate;
+    private LocalTime formFieldTime;
+    private String formFieldString;
+    private String formFieldPhoneNumber;
+
+    /**
+     * Constructs a FieldValue object
+     *
+     * @param type The type of the field.
+     */
+    public FieldValue(final FieldValueType type) {
+        this.type = type;
+    }
+
+    /**
+     * Set the map value of the field.
+     *
+     * @param formFieldMap the map value of the field.
+     *
+     * @return the FieldValue object itself.
+     */
+    public FieldValue setFormFieldMap(final Map<String, FormField> formFieldMap) {
+        this.formFieldMap = formFieldMap;
+        return this;
+    }
+
+    /**
+     * Set the list value of the field.
+     *
+     * @param formFieldList the list of the field.
+     *
+     * @return the FieldValue object itself.
+     */
+    public FieldValue setFormFieldList(final List<FormField> formFieldList) {
+        this.formFieldList = formFieldList;
+        return this;
+    }
+
+    /**
+     * Set the float value of the field.
+     *
+     * @param formFieldFloat the float value of the field.
+     *
+     * @return the FieldValue object itself.
+     */
+    public FieldValue setFormFieldFloat(final Float formFieldFloat) {
+        this.formFieldFloat = formFieldFloat;
+        return this;
+    }
+
+    /**
+     * Set the integer value of the field.
+     *
+     * @param formFieldInteger the integer value of the field.
+     *
+     * @return the FieldValue object itself.
+     */
+    public FieldValue setFormFieldInteger(final Integer formFieldInteger) {
+        this.formFieldInteger = formFieldInteger;
+        return this;
+    }
```

Comment:
How large this `Integer` can be? According to the service, we'll eventually start supporting 64-bits integers here (not a concern for Preview 3, I think).

Reply:
I think that should change the generated code for the service to start sending in a `Long` value so will defer doing this with the service update.